.github/workflows/test-deployer-code.yaml

```diff
@@ -31,4 +31,4 @@ jobs:
           python -m pip install -r requirements.txt
       - name: Run tests
         run: |
-          python -m pytest -vvv --color=always
+          python -m pytest -vvv --color=yes
```

Comment:
Aaargh woops, I always mistake `git diff --color=always` with `pytest --color=yes`...

In this case, I even did a manual check to verify I got it right --- the issue was that i verified that `always` should be used for `git diff`, not for `pytest` ^^